peps/pep-0694.rst

```diff
@@ -1023,11 +1013,9 @@ Can I use the upload 2.0 API to reserve a project name?
 Yes!  If you're not ready to upload files to make a release, you can still reserve a project
 name (assuming of course that the name doesn't already exist).
 
-To do this,
-:ref:`create a new Publishing Session <publishing-session-create>`,
-then :ref:`publish the session <publishing-session-completion>` without uploading any files.
-While the ``version`` key is required in the JSON body of the create session request,
-you can simply use the placeholder version number ``"0.0.0"``.
+To do this, :ref:`create a new publishing session <publishing-session-create>`, then :ref:`publish the session
+<publishing-session-completion>` without uploading any files.  While the ``version`` key is required in the
+JSON body of the create session request, you can simply use the placeholder version number ``"0.0.0"``.
```

Comment:
Sorry that I didn't notice this before, but is `"0.0.0"` somewhat special here?

Reply:
Proposed change:

> To do this, :ref:`create a new publishing session <publishing-session-create>`, then :ref:`publish the session
<publishing-session-completion>` without uploading any files.  While the ``version`` key is required in the
JSON body of the create session request, you can simply use a placeholder version number such as
``"0.0.0a0"``.

Reply:
That's a great question.  I think not, given that the scenario described includes no uploaded artifacts.  It's simply a name reservation.  I don't want to make `version` optional or add a different workflow just for name reservation, although we could do that in the future.